index.markdown

```diff
@@ -1,8 +1,17 @@
 ---
-layout: default
+# layout: default #change layout to 2021 to see updates
+layout: 2021
```

Comment:
toggle the # comments here so that when this goes out, we're preserving the 'default' layout